data/human/news.txt

```diff
@@ -2150,3 +2150,164 @@ news "graffiti on spaceport wall"
+		"dispurse. "
+		"abandon their objective. "
+		"come quietly. "
+	word
+		"The Guerrillas "
+	word
+		"bluster "
+		"saber-rattle "
+	word
+		"for a bit in response "
+	word
+		"but it all ends with them dispersing."
+		"and their opposition backs down."
+
+phrase "A Light Ship"
+	word
+		"A Sparrow"
+		"An Enforcer"
+		"A Shuttle"
+		"A Heavy Shuttle"
+		"A Star Barge"
+		"A Berserker"
+		"A Fury"
+		"A Hawk"
+		"A Wasp"
+
+phrase "Pronoun"
+	word
+		"his"
+		"hers"
+		"their"
+
+phrase "Careless Ends Well"
+	word
+		"turns around to reenter their ship, "
+	word
+		"as it's struck by a wave, nudging it off the platform. The captain then moves the ship to a lowerable pad."
+		"moving it to a lowerable pad before it could be caught by a wave."
+
+phrase "Careless Ends Poorly"
+	word
+		"ignores Spaceport Control and heads to the bar. The ship"
+		"argues with Control about their landing. The ship"
+		"turns around to reenter their ship, however it"
+	word
+		"is "
+	word
+		"struck "
+		"caught "
+	word
+		"by a wave and "
+	word
+		"swept into the sea."
+		"pushed off the pad."
+
+news "Careless Landing"
+	location
+		planet Jakobsen
+name
+	"Recklessness"
+message
+	phrase 
+		"A Light Ship"
+	word
+		" lands on a pad meant for heavier ships. Spaceport Control is immediately on the captain to move ${Pronoun} ship. The captain "
+	phrase
+		"Careless Ends Well"
+		"Careless Ends Poorly"
+
+phrase "CCOR Song Description"
+	word
+		"a song about "
+	word
+		"community, everyone pitching in where they can"
+		"the tyranny of a man name Bailey, until he was overthrown"
+		"CCOR, praising their acomplishments"
+		"resistance, oposition to the current planetary administration"
+
+phrase "Worker's Demands"
+	word
+		"safety regulations"
+		"increased pay"
+		"restoration of the CCOR"
+		"justice against planetary security brutality"
+		"immediate repositioning of the spaceport"
+
+news "Discontent"
+	location
+		planet Bailey
+name
+	"Conflict"
+message
+	word
+		"Allegations that planetary authorities will no longer move the spaceport has provoked "
+	word
+		"large scale "
+		"mass "
+		"medium "
+		"scattered "
+	word
+		"riots "
+		"protests %"
+	word
+		"the rioters are "
+	word
+		"singing, %{CCOR Song Description}. "
+		"chanting, a list of their demands including ${Worker's Demands}. "
+		"sizing up the planetary security forces. "
+	word
+		"Security eventually steps in "
+	word
+		"however they're pushed back. "
+		"but they're routed. "
+		"they disperse the demonstrators. "
+		"the demonstrators scatter but don't appear diswaded."
+	replace
+		"% the rioters" "the protesters"
+		"%" ""
+
+news "Clandestine"
+	location
+		planet Alvorada
+name
+	"Observation"
+message
+	word
+		"You can't escape the feeling of being watched it's "
+	word
+		"probably the security cameras, most of them are still running. "
+		"feels like there's eyes around every corner. "
+		"the pirates, a small number of them seem to be watching you. "
+	word
+		"The 
```

Comment:
Why is this here?

Reply:
This is what I meant about Alvorada's news being incomplete. I wanted to add more but writer's block struck and couldn't figure out how to continue this one.

Reply:
If it's incomplete and you're planning to add more to this PR, you might want to mark it as a draft.

Reply:
Yeah. Converted to draft.